### ISSUE TITLE
ec.HasError undefined (type *executionContext has no field or method HasError)

### ISSUE THREAD

**Issue by tkallioras** [99designs/gqlgen#305]
I am running gqlgen 0.4.4 and when I try to compile I get error:

`ec.HasError undefined (type *executionContext has no field or method HasError)
`

Is there something I am missing?

**Comment by vektah** [comment]
You're `gqlgen` binary is using master, but the code in your repo is using `0.4.4`

https://gqlgen.com/getting-started/#finishing-touches describes how to keep the two in sync (don't use a globally installed binary)

**Comment by vektah** [comment]
The fix for now is either roll back to 0.4.4:
```go
cd $GOPATH/src/github.com/99designs/gqlgen
git checkout v0.4.4
go install
```
or run gqlgen out of vendor
```
gorunpkg github.com/99designs/gqlgen # instead of running gqlgen directly
```

I think when go module support lands this fixes itself.

**Comment by Southclaws** [comment]
Hi, running into this (6 years later!) same issue, versions are locked for both imports and generator:

```go.mod
github.com/99designs/gqlgen v0.17.44
```

```yml
go run github.com/99designs/gqlgen@v0.17.44 generate
```

It seems to happen when trying to bind an enum to a Go type.
